### PR TITLE
expanded teacher and learner introductions

### PR DESCRIPTION
**Pull request by raynamharris**
Expand the section on teacher introductions and student introductions. cc @tracykteal

### CODE REVIEW

File: _episodes/01-welcome.md [hunk 1]
```diff
@@ -34,7 +34,14 @@ instructor training.  We're very pleased to have you with us.
...
 >
-> Each trainer can introduce themselves. 
+> To begin your class, each instructor should give a brief introduction that will convey 1) their capacity to teach the material, 2) accessibility/approachability, 3) desire for student success, and 4) enthusiasm. Tailor your introduction to the students’ skill level so that you convey competence (without seeming too advanced) and demonstrate that you can relate to the students. Throughout the workshop, continually demonstrate that you are interested in student progress and that you are enthusiastic about the topics.
+{: .discussion}
+
+> ## Today's Learners
+>
+> 1. Everyone should add their name to the Etherpad (this can be done while setting up before the start of the class). 
+> 1. Students should also introduce themselves (preferably verbally). It is also good for everyone at a given site to know who all is in the group.
+> 1. As part of their instroduction, ask students to 1) explain thier work in 3 works and 2) say something they are proud of (not necessarily related to research or teaching).
```
Comment (lexnederbragt) [inline _episodes/01-welcome.md]: 3 works --> 3 words

**Comment by lexnederbragt** [comment]
students --> learners OR trainees (?)

**Comment by raynamharris** [comment]
Ah yes. I prefer learners. Good catch @lexnederbragt

**Comment by ChristinaLK** [comment]
It feels a bit funny to put text like "Tailor your introduction to the learners’ skill level so that you convey competence (without seeming too advanced)" in a learner-facing page, but until we have another plan/method for instructional notes, I'm okay with this where it is.

**Comment by ErinBecker** [comment]
@ChristinaLK I agree that it seems a bit funny, but it's also a good example of eating our own dogfood. We tell the instructors to do all these things when they teach, so it shouldn't be too surprising for the learners at TTT events that their Trainers do this too!

**Pull request merged**